Mage/src/main/java/mage/abilities/effects/common/RegenerateTargetEffect.java

```diff
@@ -63,4 +68,20 @@ public String getText(Mode mode) {
         }
         return "regenerate " + getTargetPointer().describeTargets(mode.getTargets(), "that creature");
     }
+
+    @Override
+    public boolean hasHint() {
+        return true;
+    }
+
+    StaticHint hint = new StaticHint("{this} is regenerating.");
```

Comment:
I'm a bit confused about this one. Isn't there already a hint for regeneration shields?

Reply:
There is - I didn't notice since it's only in `RegenerateSourceEffect`. Let me remove this bit.